app/experimenter/nimbus-ui/src/components/PageRequestReview/index.tsx

```diff
@@ -18,30 +20,67 @@ import {
 import { updateExperiment_updateExperiment as UpdateExperiment } from "../../types/updateExperiment";
 import AppLayoutWithExperiment from "../AppLayoutWithExperiment";
 import Summary from "../Summary";
-import FormLaunchDraftToPreview from "./FormLaunchDraftToPreview";
-import FormLaunchDraftToReview from "./FormLaunchDraftToReview";
+import DraftStatusOperations from "./DraftStatusOperations";
 import FormLaunchPreviewToReview from "./FormLaunchPreviewToReview";
 
 const PageRequestReview = ({
+  /* istanbul ignore next - only used in tests & stories */
   polling = true,
+  /* istanbul ignore next until EXP-1055 & EXP-1062 done */
+  isLaunchRequested = false, // new experiment property
+  /* istanbul ignore next until EXP-1055 & EXP-1062 done */
+  isLaunchApproved = false, // new experiment property
+  /* istanbul ignore next until EXP-1055 & EXP-1062 done */
+  launchRequestedByUsername = "", // new experiment property
+  /* istanbul ignore next until EXP-1055 & EXP-1062 done */
+  currentUsername = "", // new user ID / email property
+  /* istanbul ignore next until EXP-1055 & EXP-1062 done */
+  currentUserCanApprove = false, // new user permission property
 }: {
   polling?: boolean;
+  // TODO EXP-1062: temporary page-level props, should be replaced by API data for experiment & current user
```

Comment:
I haven't dug into the back-end issues in particular yet, but the UI implies that at least these pieces of data will be necessary to drive it. Assuming a few of them will come along with the experiment. Others - i.e. user details - should probably some with NimbusConfig?

Reply:
Hrmmm yeah, agreed we need these pieces of data. Sending the current user data down with `NimbusConfig` seems right to me since we rerequest the experiment data so often. cc @jaredlockhart though